http/logical.go

```diff
@@ -90,6 +91,10 @@ func buildLogicalRequest(core *vault.Core, w http.ResponseWriter, r *http.Reques
 
 func handleLogical(core *vault.Core, dataOnly bool, prepareRequestCallback PrepareRequestFunc) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// Bubble up CORS preflight requests to the CORS handler.
```

Comment:
I'll need to test some things and reply more in depth later. I believe I was encountering errors, because a preflight won't have any data associated with it.